<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolarCam</class>
 <widget class="QWidget" name="PolarCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PolarCam</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="Parent_Frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="ImageFeed_Frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="cameraFeedLabel">
           <property name="maximumSize">
            <size>
             <width>1600</width>
             <height>950</height>
            </size>
           </property>
           <property name="text">
            <string>OpenCV Image Here</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Controls_Frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="startCameraBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>161</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Start Camera Preview</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>380</y>
           <width>281</width>
           <height>321</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="ConfigureSequenceBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Configure Sequence</string>
          </property>
         </widget>
         <widget class="QPushButton" name="startSequenceBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Start Sequence Mode</string>
          </property>
         </widget>
         <widget class="QPushButton" name="resetSequenceBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Reset Sequence</string>
          </property>
         </widget>
         <widget class="QPushButton" name="captureSeqBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Capture</string>
          </property>
         </widget>
         <widget class="QPushButton" name="viewCaptureSeqBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>220</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>View Capture Sequence</string>
          </property>
         </widget>
         <widget class="QLabel" name="SequenceModeLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>121</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Sequence Mode</string>
          </property>
         </widget>
         <widget class="QPushButton" name="saveSeqToFileBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>250</y>
            <width>171</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Save Sequence To File</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="stopCameraBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>161</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop Camera Preview</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>281</width>
           <height>181</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QSlider" name="gainSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>201</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="shutterSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>201</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>71</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Gain (db)</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>81</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Shutter (us)</string>
          </property>
         </widget>
         <widget class="QLabel" name="gain_db_textlabel">
          <property name="geometry">
           <rect>
            <x>116</x>
            <y>10</y>
            <width>81</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
         <widget class="QLabel" name="shutter_us_textlabel">
          <property name="geometry">
           <rect>
            <x>116</x>
            <y>70</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="GainAuto_chkBox">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="shutterAuto_chkBox">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>80</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>81</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>FPS</string>
          </property>
         </widget>
         <widget class="QSlider" name="frameRateSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>201</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QCheckBox" name="frameRateAuto_chkbox">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>140</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Auto</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="fps_textLabel">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>130</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="captureImg_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>161</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="view_captureCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>View Capture</string>
         </property>
        </widget>
        <widget class="QPushButton" name="sendImageToOCR_Btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>161</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Send Image to OCR</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
